ocd_backend/es.py

```diff
@@ -4,6 +4,7 @@
 
 
 def setup_elasticsearch(host=ELASTICSEARCH_HOST, port=ELASTICSEARCH_PORT):
-    return Elasticsearch([{'host': host, 'port': port}])
+    return Elasticsearch([{'host': host, 'port': port}], http_auth=('elastic', 'changeme'))
```

Comment:
The current version of the used elasticsearch docker image sets authentication by default. They offer ways to change it, however I did not find a neat way to disable auth. Since elastic isn't exposed to the public the passwords here pose a slight security risk, which would have been the same as without any password.